src/clp_ffi_js/ir/StreamReader.hpp

```diff
@@ -264,7 +264,14 @@ auto StreamReader::generic_decode_range(
         auto const& log_level = log_event_with_filter_data.get_log_level();
 
         EM_ASM(
```

Comment:
+ ditto (see above about prop rename and alphabetization)
+ it seems we can reduce one level of indent in the EM_ASM block
```
        EM_ASM(
            {
                Emval.toValue($0).push({
                    "logEventNum": $1,
                    "logLevel": $2,
                    "message": UTF8ToString($3),
                    "timestamp": $4,
                });
            },
            results.as_handle(),
            log_event_idx + 1,
            log_level,
            log_event_to_string(log_event).c_str(),
            timestamp
        );
```

Reply:
Somehow the indent was not applied on my mac when i ran `task:lint-cpp-full`. Let me double check